ui/apps/platform/cypress/integration/compliance/Compliance.helpers.js

```diff
@@ -121,7 +119,11 @@ function scanCompliance() {
         { timeout: 20000 }
     );
 
-    cy.get(scanButton).should('not.have.attr', 'disabled');
```

Comment:
The code was changed so that this button is never disabled, so these checks are unnecessary.